MAUI/DataGrid/Column-types.md

```diff
@@ -0,0 +1,916 @@
+---
+layout: post
+title: Column Types in .NET MAUI DataGrid control | Syncfusion
+description: Learn here all about Column Types support in Syncfusion .NET MAUI DataGrid (SfDataGrid) control and more.
+platform: MAUI
+control: SfDataGrid
+documentation: UG
+---
+
+# Column Types in .NET MAUI DataGrid (SfDataGrid)
+
+The SfDataGrid contains different types of columns. The functionalities of the column can be implied by its name. Based on the requirements any column can be used. 
+
+The following table describes the types of columns and its usage:
+
+<table>
+<tr>
+<th>Column Type</th>
+<th>Renderer</th>
+<th>Description</th>
+</tr>
+<tr>
+<td>{{'[DataGridTextColumn]'| markdownify }}</td>
+<td>{{`DataGridTextBoxCellRenderer` }}</td>
+<td>To display string or numbers in each row.</td>
+</tr>
+<tr>
+<td>{{'[DataGridCheckBoxColumn]'| markdownify }}</td>
+<td>{{DataGridCheckBoxCellRenderer }}</td>
+<td>To display Checkbox in each row.</td>
+</tr>
+<tr>
+<td>{{'[DataGridImageColumn]'| markdownify }}</td>
+<td>{{`DataGridImageCellRenderer`}}</td>
+<td>To display an image in each row.</td>
+</tr>
+<tr>
+<td>{{'[DataGridTemplateColumn]'| markdownify }}</td>
+<td>{{'[DataGridCellTemplateRenderer]'| markdownify }}</td>
+<td>To customize the column based on the requirements.</td>
+</tr>
+<tr>
+<td>{{'[DataGridNumericColumn]'| markdownify }}</td>
+<td>{{'[DataGridNumericCellRenderer]'| markdownify }}</td>
+<td>To display a numeric data.</td>
+</tr>
+<tr>
+<td>{{'[DataGridDateColumn]'| markdownify }}</td>
+<td>{{'[DataGridDateCellRenderer]'| markdownify }}</td>
+<td>To display the date and time value.</td>
+</tr>
+</table>
+
+
+## DataGridColumn
+
+The `DataGridColumn` is the base column types of all columns. Hence its properties are used by all the columns. The following sub-sections explain the properties and customizations of DataGridColumn:
```

Comment:
The `DataGridColumn` is the base column type of all columns, hence its properties are used by all the columns. The following sub-sections explain the properties and customizations of DataGridColumn: